.github/workflows/lint-and-test-matrix.yml

```diff
@@ -51,15 +51,22 @@ jobs:
         python -m pip install --upgrade pip
         pip install -e '.[all]'
 
-    - name: Run Tests with Coverage
+    - name: Run Tests
       run: |
-        pip install coverage pytest pytest-cov
-        coverage run -m pytest --maxfail=1 --disable-warnings -q
-        coverage report -m
-        coverage html
+        pip install pytest pytest-cov
```

Comment:
Are these not covered by https://github.com/wmalgadey/PyTado/blob/master/pyproject.toml#L34

when `pip install -e '.[all]'` is executed above?